sql-join/where-are-they.sql

```diff
@@ -0,0 +1,15 @@
+select "customers"."firstName",
+"customers"."lastName",
+"payments"."amount",
+"payments"."customerId",
```

Comment:
We don't need any customer info for this one, just full addresses.

![image](https://user-images.githubusercontent.com/7432943/130288957-50ba13a9-d35d-45f9-94e2-fe2425518b1b.png)